src/components/Product/Product.js

```diff
@@ -0,0 +1,26 @@
+import Image from 'next/image';
+import { Container, ImageWrapper, Article, ProductDetail } from './StyledProduct';
+import EmailButton from '../Button/StyledButton';
+
+export default function Product({ products }) {
+	return (
+		<Container>
+			<ImageWrapper>
+				<Image
+					src="/test.jpg"
+					alt="dress"
+					width={3456}
+					height={5184}
+					layout="fill"
+					objectFit="cover"
+				/>
+			</ImageWrapper>
+			<Article>
```

Comment:
I am confused that the "Article" styled component is a div and not an article element.